src/views/ViewView.vue

```diff
@@ -89,6 +91,25 @@ export default defineComponent({
       updateLoadingStdErr,
       updateLoadingStdWin,
     } = useLoading();
+
+
+     // TODO: generalize logic more + page to allow multiple calls/groups
+     // based on different values ('open', 'closed', etc.)
+     function filterInOpenStatusNFT (arr: Array<INFT>) {
+       // Takes in an array of NFTs & filters to just "open" ones
+       // Assumes metatadataExternal attributes are fixed where index 1 {} is trait-type status
+       let arr_keep: Array<INFT> = []
+       for (var n of arr){
+          if (n.metadataExternal.attributes.length >= 2){
```

Comment:
same comment as earlier. We should look up the attribute by the attribute type explictly instead of assuming its always in a specific order.

Reply:
e.g. attributes.get("ticket_status")